### PR TITLE
GetValue implementation for unsigned integers #1784

### PR DESCRIPTION
**Pull request by pdet**
This PR implements GetValue for unsigned integers and changes GetValue of uintptr to a GetPointer function,

### CODE REVIEW

**Comment by hannesmuehleisen** [comment]
I think the R failure is due to the PointerValue change as well, since it tests 32 bit

**Comment by pdet** [comment]
@hannesmuehleisen I don't think the getvalue for uintptr is used on R is it?

**Comment by hannesmuehleisen** [comment]
It probably is in the data frame scans

File: tools/rpkg/src/duckdbr.cpp [hunk 1]
```diff
@@ -908,7 +908,7 @@ struct DataFrameScanFunction : public TableFunction {
 	                                                    vector<string> &input_table_names,
 	                                                    vector<LogicalType> &return_types, vector<string> &names) {
 		RProtector r;
-		SEXP df((SEXP)inputs[0].GetValue<uintptr_t>());
+		SEXP df((SEXP)inputs[0].GetPointer());
```
Comment (pdet) [inline tools/rpkg/src/duckdbr.cpp:911]: @hannesmuehleisen this one?

Comment (hannesmuehleisen) [reply tools/rpkg/src/duckdbr.cpp:911]: Yup

**Comment by hannesmuehleisen** [comment]
thanks!

**Pull request merged**